decent-dao/fractal-interface
# add Proposal Details for create proposal page

## Description
Added Proposal Detail info to the Create Proposal page per the design documents. 
Also made some fixed to the GoveranceInfo components as they load very similar data.


## Notes
NOTE: There are still some loading issue in which loading a favorite just show the loading bars, I believe that is fixed in an open PR. But refresh and the data will load correctly.

## Issue / Notion doc (if applicable)

## Testing
- Open a DAO
- From the Proposals Page choose Create Proposal.
- Verify details displayed in the panel on the right are correct for the currently selected DAO.


## Screenshots (if applicable)
![image](https://user-images.githubusercontent.com/99622829/206302939-f99db471-01f7-4bf5-a49a-a1bcf9eec96a.png)
![image](https://user-images.githubusercontent.com/99622829/206303052-f2f2a15c-7bfe-4687-98f0-9898ea156163.png)
